src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,84 @@
+package frc.robot.subsystems;
+
+import static edu.wpi.first.wpilibj2.command.Commands.waitSeconds;
+
+import com.chopshop166.chopshoplib.logging.LoggedSubsystem;
+import com.chopshop166.chopshoplib.motors.CSSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.maps.subsystems.ShooterMap;
+import frc.robot.maps.subsystems.ShooterMap.Data;
+
+public class Shooter extends LoggedSubsystem<Data, ShooterMap> {
+
+    private final double RELEASE_SPEED = 1.00;
+    private final double AMP_SPEED = .50;
+    private final double SLOW_SPEED = .25;
+    private final double HALF_SPEED = .50;
+    private final double THREE_QUARTER = .75;
```

Comment:
Can we make this an enum and then have a function that takes the enum and gets the shooter up to speed instead of having separate functions for each speed here?